play/src/pages/pregame/SelectAvatar.tsx

```diff
@@ -136,7 +137,13 @@ export default function SelectAvatar({
           <Typography variant="h2" sx={{ textAlign: 'center' }}>
             {`${firstName} ${lastName}`}
           </Typography>
-          <GamePlayButtonStyled onClick={handleAvatarSelectClick}>
+          <GamePlayButtonStyled
+            onClick={() => {
+              handleAvatarSelectClick();
+              setIsButtonPressed(true);
```

Comment:
Is `handleAvatarSelectClick` async? Wondering if `setIsButtonPressed` should be called before this or not